src/Compiler/ReflectionPrinter.cpp

```diff
@@ -24,8 +24,18 @@ void ReflectionPrinter::PrintReflection(const Reflection::ReflectionData& reflec
     output_ << R_CodeReflection() << ':' << std::endl;
     indentHandler_.IncIndent();
     {
+        //vector<uniform> -> vector<string>
```

Comment:
There are multiple overloaded versions of the `PrintReflectionObjects` function, which I recommend to overload again instead of creating temporary `vector<string>` containers.